### DISCUSSION TITLE
Entities in HA becomes N/A

### DISCUSSION THREAD

**Discussion by HK-Moebius** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#16] [General]
Hi there,

Since a few days the entities sporadic are N/A in Home Assistant. This happends while sunshine and producing energy.

**Comment by andrewoe** [comment]
same here; But fixed that with the firmware upgrade to 1.7.1

**Reply by HK-Moebius** [reply]
Yesterday I checked for FW update and the latest was already installed.
